### PR TITLE
AWS How To pages

### PR DESCRIPTION
**Pull request by mehzer**
Added a section under /aws.
Suggest that we amend the footer 'AWS' to link to /aws

Please especially check code, repo links and the like.

### CODE REVIEW

File: aws/iam.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: "How to create an AWS Identity and Access Management service with Pulumi"
```
Comment (justinvp) [inline aws/iam.md]: Nit: Another case where it might be worth including IAM in parentheses.

File: aws/athena.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+library: "@pulumi/aws"
+brand: "aws"
+og:
+    description: "Use Pulumi to code, deploy, and manage cloud, serverless, and container apps and infrastructure"
+    image: "/images/service/aws-athena.png"
+---
+<!-- Links -->
+[AWS here]: https://aws.amazon.com/athena/
+[Reference docs]: ../reference/aws.html
+[Example code]: https://github.com/pulumi/examples/tree/master/aws-js-s3-folder
```
Comment (justinvp) [inline aws/athena.md]: This link isn't used and links to an s3 example. Applies to some of the other services as well.

File: aws/kinesis.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+</div>
+
+
+
+## Create an {{ page.cloud }} {{ page.service }} resource using `{{ page.library }}`
+
+The `{{ page.library }}` library enables fine-grained control over the {{ page.cloud }} {{ page.service }} resource meaning it can be coded, deployed, and managed entirely in code. 
+
+```javascript
+const stream = new aws.kinesis.Stream("mystream", {
```
Comment (justinvp) [inline aws/kinesis.md]: This snippet is missing the `const aws = require("@pulumi/aws");`

File: aws/sns.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+</div>
+
+
+
+## Create an {{ page.cloud }} {{ page.service }} resource using `{{ page.library }}`
+
+The `{{ page.library }}` library enables fine-grained control over the {{ page.cloud }} {{ page.service }} resource meaning it can be coded, deployed, and managed entirely in code. 
+
+```javascript
+const topic = new aws.sns.Topic("mytopic");
```
Comment (justinvp) [inline aws/sns.md]: This snippet is missing the `const aws = require("@pulumi/aws");`

File: aws/sqs.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+</div>
+
+
+
+## Create an {{ page.cloud }} {{ page.service }} resource using `{{ page.library }}`
+
+The `{{ page.library }}` library enables fine-grained control over the {{ page.cloud }} {{ page.service }} resource meaning it can be coded, deployed, and managed entirely in code. 
+
+```javascript
+const queue = new aws.sqs.Queue("myqueue");
```
Comment (justinvp) [inline aws/sqs.md]: This snippet is missing the `const aws = require("@pulumi/aws");`

File: aws/ecr.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+</div>
+
+
+
+## Create an {{ page.cloud }} {{ page.service }} resource using `{{ page.library }}`
+
+The `{{ page.library }}` library enables fine-grained control over the {{ page.cloud }} {{ page.service }} resource meaning it can be coded, deployed, and managed entirely in code. 
+
+```javascript
+const repository = new aws.ecr.Repository("myrepository");
```
Comment (justinvp) [inline aws/ecr.md]: This snippet is missing the `const aws = require("@pulumi/aws");`

File: aws/index.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+</div>
+
+## Multi-cloud development with `@pulumi\cloud`
+
+For a higher-level abstraction, and greater portability, the [`@pulumi\cloud-aws`](../reference/cloud.html) library provides a set of classes that enable productive development for any cloud. When instantiated for AWS, code is adapted to use the available services in AWS, and the supporting services necessary to deploy them (e.g. IAM roles).
+
+```javascript
+var cloud = require("@pulumi/cloud-aws");
+
+//Create RestAPI and Lambda Functions
```
Comment (justinvp) [inline aws/index.md]: Nit: Space after `//`

**Comment by mehzer** [comment]
Thanks - all feedback addressed, and commit just made.

**Pull request merged**